### ISSUE TITLE
Some improvements / requests

### ISSUE THREAD

**Issue by windows-server-2003** [d0k3/GodMode9#299]
1. Screenshots path
Now, they are saved at the root of the SD. It dirties the root so they should be saved in /gm9/screenshots.

2. Save screenshots in sysnand
It's nice to save screenshots to 1:/rw/gm9/screenshots if SD is not mounted.

3. PCX image viewer
PCX image viewer via the file handler menus will be a good addition.

4) B button handling in some menus
Pressing B in some submenus(like file attributes editor) will leads us back to the file browser.
But it's more useful to go back to the previous menu(in the case, the file handler menu) instead.

**Comment by urherenow** [comment]
Screenshots to nand? I'd be surprised if it was even considered. They take up too much space. Bad idea IMHO.

**Comment by windows-server-2003** [comment]
A screenshot file takes about 500KB.
100(screenshots will only be saved to 1: if SD isn't mounted so more than enough) -> about 50MB
The freespace of 1: is often more than 200MB.

**Comment by SirNapkin1334** [comment]
Wasn't d0k3 going on about writing to the NAND as little as possible on the thread? I'd say screenshots to NAND would be a waste of a write. Though, scripts/support under 1:/rw/gm9/ would be smart IMO.

**Comment by windows-server-2003** [comment]
>Wasn't d0k3 going on about writing to the NAND as little as possible on the thread? I'd say screenshots to NAND would be a waste of a write. Though, scripts/support under 1:/rw/gm9/ would be smart IMO.

I said why don't you write it to a specified sector of NAND. That will write to the physically same location on Temp, so it was a bad idea.
But this time, it will be written as file and written to sectors that have been written fewer times(maybe I'm wrong but anyways won't be written at the same sectors everytime).

**Comment by SirNapkin1334** [comment]
Just a random question, can you close other people's issues like d0k3?

**Comment by windows-server-2003** [comment]
No, only people who have write access to the repository(which I don't have) and the opener can.
(I think no one other than d0k3 have the permission to write)

**Comment by d0k3** [comment]
`1:/gm9` already works as a replacement for everything `0:/gm9` does - except the `0:/gm9/out` folder. And, yes, a separate folder for screenshots is a good idea. I'll look into that.

The only person other than me that has write access here is @Wolfvak.

**Comment by SirNapkin1334** [comment]
@windows-server-2003 then how do you push commits? Or is it because d0k3 merged your fork?

**Comment by urherenow** [comment]
Anyone can push a commit. It only gets merged when accepted by someone with write access. Now stop this. Github is not a forum.

**Comment by windows-server-2003** [comment]
@SirNapkin1334
Yes, I forked d0k3/GodMode9 to windows-server-2003/GodMode9 and pushed commits to my one.
Then, d0k3 merged mine to his.

**Comment by d0k3** [comment]
Explanation: I changed the default screenshot directory to `0:/gm9/out`. Tbh, I don't see a use case where writing to NAND makes sense yet, so I left that for now (feel free to get back about that). NAND writes should be kept to a minimum in any case, so a default output file on NAND is totally out of question.

I also added a (somewhat limited) PCX viewer. It is limited by the size of one of the screens and it will only display 8bpp PCX (aka the splash format).

As for B button handling - it is assumed once you did what you intended, once an action finishes, you want out of the menu. The handling for the file attribute editor is fine imho. Feel free to get back about that, too.
